Makefile

```
@@ -76,5 +76,8 @@ endif
 test-py:
 	pytest . --ignore=tests/integration --ignore=scripts/2011 --ignore=infogami --ignore=vendor --ignore=node_modules
 
+test-i18n:
+	$(PYTHON) ./scripts/i18n-messages validate_production
```

To make this simpler, let's update `validate` to handle multiple languages;

```suggestion
	$(PYTHON) ./scripts/i18n-messages validate en fr de
```

That way we don't need to add another command to the `i18n-messages`, and we also make `validate` behave like the other languages.